main.js

```diff
@@ -1,68 +1,12 @@
-var level = require("level");
-var db  = level("./student-db", { valueEncoding: 'json'});
-var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probationary'];
+const level = require('level');
 
-const prompt = require('prompt-sync')({sigint: true});
+(function main() {
+    const db = connectToDatabase('./Student-db');
+}());
```

Comment:
you may want to declare main function as async , in preparation for other tasks , allowing you to use await like:
```suggestion
(async function main() {
    const db = connectToDatabase('./Student-db');
}());
```